odoo/addons/test_new_api/tests/test_new_fields.py

```diff
@@ -250,6 +250,13 @@ def test_10_computed_stored_x_name(self):
         """)
         # setting up models should not crash
         self.registry._setup_models__(self.cr)
+    
+    def test_10_context_dependent_related(self):
```

Comment:
IMO, it is worth it to really test the expected behavior of  `container_context_id`/`container_context_name`. Don't you think ?